README.md

```diff
@@ -18,7 +18,7 @@ Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instruction
 
 ## Configuration
 
-The ad adapter model can be built based on either the Final URL Performance Report or the Criteria Performance Report. You can choose which by setting the `google_ads__ad_adater_source` variable to either `criteria` or `url`.
+We can build the ad adapter model based on either the Final URL Performance Report or the Criteria Performance Report. Choose which by setting the `google_ads__ad_adapter_source` variable to `url` or `criteria` respectively.
```

Comment:
@fivetran-kristin If possible, we should provide links to the documentation for the Final URL and Criteria performance reports.